expression/builtin_time_vec_test.go

```diff
@@ -123,7 +123,12 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.TimestampLiteral: {},
 	ast.SubDate:          {},
 	ast.AddDate:          {},
-	ast.SubTime:          {},
+	ast.SubTime: {
+		{
+			retEvalType:   types.ETString,
+			childrenTypes: []types.EvalType{types.ETString, types.ETString},
```

Comment:
Please add some test for legal string that can convert to time.

Reply:
We can use selectStringGener to generate the string we want. Random string can't convert to time and the results will be null.

Reply:
timeStrGener can use to generate random time string.